### PR TITLE
Make pw in mail optional when user created by admin

### PR DESCRIPTION
**Pull request by sovainfo**
Redo of #10462

### CODE REVIEW

File: administrator/language/en-GB/en-GB.plg_user_joomla.ini [hunk 1]
```diff
@@ -8,11 +8,14 @@ PLG_USER_JOOMLA_FIELD_AUTOREGISTER_DESC="Automatically create Registered Users w
 PLG_USER_JOOMLA_FIELD_AUTOREGISTER_LABEL="Auto-create Users"
 PLG_USER_JOOMLA_FIELD_FORCELOGOUT_DESC="Set to No to disable this."
 PLG_USER_JOOMLA_FIELD_FORCELOGOUT_LABEL="Force Logout for all Sessions?"
+PLG_USER_JOOMLA_FIELD_INCLUDEPASSWORD_DESC="Include the password when sending a mail to the created user."
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.plg_user_joomla.ini]: I think k this "an email" not "a mail"

File: plugins/user/joomla/joomla.xml [hunk 2]
```diff
@@ -43,6 +43,18 @@
...
+					class="btn-group btn-group-yesno"
+					default="1"
+					showon="mail_to_user:1"
+					label="PLG_USER_JOOMLA_FIELD_INCLUDEPASSWORD_LABEL"
+					description="PLG_USER_JOOMLA_FIELD_INCLUDEPASSWORD_DESC"
+				>
+					<option value="1">JYES</option>
+					<option value="0">JNO</option>
+				</field>
+
```
Comment (wojsmol) [inline plugins/user/joomla/joomla.xml]: see http://joomla.github.io/coding-standards/?coding-standards/chapters/xml.md

File: plugins/user/joomla/joomla.xml [hunk 3]
```diff
@@ -45,11 +45,11 @@
 
 				<field name="include_password_in_mail_to_user"
```
Comment (wojsmol) [inline plugins/user/joomla/joomla.xml]: name to new line

File: plugins/user/joomla/joomla.xml [hunk 4]
```diff
@@ -45,11 +45,11 @@
...
 					type="radio"
+					label="PLG_USER_JOOMLA_FIELD_INCLUDEPASSWORD_LABEL"
+					description="PLG_USER_JOOMLA_FIELD_INCLUDEPASSWORD_DESC"
 					class="btn-group btn-group-yesno"
 					default="1"
 					showon="mail_to_user:1"
-					label="PLG_USER_JOOMLA_FIELD_INCLUDEPASSWORD_LABEL"
-					description="PLG_USER_JOOMLA_FIELD_INCLUDEPASSWORD_DESC"
 				>
```
Comment (wojsmol) [inline plugins/user/joomla/joomla.xml]: one tab more

File: plugins/user/joomla/joomla.xml [hunk 5]
```diff
@@ -43,6 +43,18 @@
 					<option value="0">JNO</option>
 				</field>
 
+				<field name="include_password_in_mail_to_user"
```
Comment (wojsmol) [inline plugins/user/joomla/joomla.xml]: name to new line

File: plugins/user/joomla/joomla.xml [hunk 6]
```diff
@@ -43,6 +43,18 @@
...
+					type="radio"
+					label="PLG_USER_JOOMLA_FIELD_INCLUDEPASSWORD_LABEL"
+					description="PLG_USER_JOOMLA_FIELD_INCLUDEPASSWORD_DESC"
+					class="btn-group btn-group-yesno"
+					default="1"
+					showon="mail_to_user:1"
+				>
```
Comment (wojsmol) [inline plugins/user/joomla/joomla.xml]: one more tab

**Comment by sovainfo** [comment]
Lost interest!

**Pull request closed without merging**